includes/reader-revenue/woocommerce/class-woocommerce-sync.php

```diff
@@ -0,0 +1,306 @@
+<?php
+/**
+ * WP CLI scripts for managing WooCommerce Reader Revenue data.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Stripe Syncs.
+ */
+class WooCommerce_Sync {
+	// User roles that a customer can have.
+	const CUSTOMER_ROLES = [ 'customer', 'subscriber' ];
+
+	/**
+	 * The final results object.
+	 *
+	 * @var array
+	 * @codeCoverageIgnore
+	 */
+	private static $results = [
+		'processed' => 0,
+	];
+
+		/**
+		 * Initialize.
+		 *
+		 * @codeCoverageIgnore
+		 */
```

Comment:
Minor nit: docblock spacing gets weird here :)

Reply:
Whoops, fixed in 44605bc9!